src/qt/splashscreen.cpp

```diff
@@ -37,70 +38,94 @@ SplashScreen::SplashScreen(Qt::WindowFlags f, const NetworkStyle *networkStyle)
     // no window decorations
     setWindowFlags(Qt::FramelessWindowHint);
 
+    // Geometries of splashscreen
+    int width = 380;
+    int height = 460;
+    int logoWidth = 270;
+    int logoHeight = 270;
+
     // set reference point, paddings
-    int paddingLeft             = 14;
-    int paddingTop              = 470;
-    int titleVersionVSpace      = 17;
-    int titleCopyrightVSpace    = 22;
+    int paddingTop = 10;
+    int titleVersionVSpace = 25;
 
     float fontFactor            = 1.0;
+    float scale = qApp->devicePixelRatio();
 
     // define text to place
     QString titleText       = tr(PACKAGE_NAME);
-    QString versionText     = QString(tr("Version %1")).arg(QString::fromStdString(FormatFullVersion()));
-    QString copyrightText   = QString::fromUtf8(CopyrightHolders("\xc2\xA9", 2014, COPYRIGHT_YEAR).c_str());
+    QString versionText = QString::fromStdString(FormatFullVersion()).remove(0, 1);
     QString titleAddText    = networkStyle->getTitleAddText();
 
-    QFont font = GUIUtil::getFontNormal();
+    QFont fontNormal = GUIUtil::getFontNormal();
+    QFont fontBold = GUIUtil::getFontBold();
+
+    QPixmap pixmapLogo = networkStyle->getSplashImage();
+    pixmapLogo.setDevicePixelRatio(scale);
+
+    // Adjust logo color based on the current theme
+    QImage imgLogo = pixmapLogo.toImage().convertToFormat(QImage::Format_ARGB32);
+    QColor logoColor = GUIUtil::getThemedQColor(GUIUtil::ThemedColor::BLUE);
+    for (int x = imgLogo.width(); x--;) {
+        for (int y = imgLogo.height(); y--;) {
```

Comment:
Hmmm, I don't love these for loops.

Can we at do:
```
for (int x = imgLogo.width(); x > 0;) {
    x--;
...
```

Or at least
`for (int x = imgLogo.width(); x-- > 0;) {` ?

Reply:
Hmmm. IMO these loops are ok-ish but if we are going to change them I'd prefer a more classical form i.e. `for (int x = imgLogo.width(); x > 0; --x) {`